app/api/dao/mentorship_relation.py

```diff
@@ -113,36 +113,38 @@ def create_mentorship_relation(self, user_id, data):
 
     @staticmethod
     @email_verification_required
-    def list_mentorship_relations(user_id=None, accepted=None, pending=None, completed=None, cancelled=None, rejected=None):
+    def list_mentorship_relations(user_id: int = None, rel_states: List[str] = None):
```

Comment:
It's just a type hint, very helpful for developers using IDEs like PyCharm. It doesn't have any runtime impact.
Shall I remove it?

Reply:
Maybe we can mention that in comments. But leave it right now, maybe I will discuss it with the project maintainers and mentors and request changes later in case required.

Reply:
Okay, deleted :)